### ISSUE TITLE
Linux: Fedora 21 - R 3.2.1 - warning message in lesson 4 vectors - R Programming - Coursera 

### ISSUE THREAD

**Issue by Jeeppler** [swirldev/swirl#339]
Hey,

I got this message during lesson 4 vectors in the R Programming package:

> my_name <- c(my_char, "my_name")

| You are quite good my friend!

  |====================================            |  76%

| Take a look at the contents of my_name.

warning messages from top-level task callback 'mini'
Warning message:
In stri_c(..., sep = sep, collapse = collapse, ignore_null = TRUE) :
  longer object length is not a multiple of shorter object length

system: Linux 
distro: Fedora 21
R-Version: R 3.2.1

Package: swirl
Sub-selection: R Programming
Lesson Number: 4
Lesson Name: Vectors

Do you need more information?

**Comment by seankross** [comment]
This error has been reported by a few users and I'm investigating it. Did this interfere with the progression of the lesson at all?

**Comment by Jeeppler** [comment]
@seankross no, it did not interfere with the progression and I did not loose any progress I have done. It is just annoying. 

My guess is an formatting error in the text, which leads to this error.

If I can further help you let me know.

**Comment by seankross** [comment]
Just curious but what is your reasoning for thinking it's a formatting error?

**Comment by Jeeppler** [comment]
It is just an intuitive guess not based on some "real" facts. 
The guess is based on the assumption that ```stri_c``` must be an string function implemented in C or C++. I guess C. C is not really safe in terms of strings, one wrong memory access can produce a wired behaviour. 

This would be my assumption, but I am an R newbie, so don't take me too serious.

If you find out what it was let me know, I am curious too.
